model/sparql/GenericSparql.php

```diff
@@ -2243,7 +2243,11 @@ private function transformChangeListResults($result) {
             }
 
             if (isset($row->date)) {
-                $concept['date'] = $row->date->getValue();
+                try {
+                    $concept['date'] = $row->date->getValue();
+                } catch (Exception $e) {
+                    //don't do anything to malformed dates e.g. 1986-21-00
```

Comment:
There should be a unit test that checks that omitting the `$concept['date']` value in the case of malformed dates - as is done here - doesn't cause further problems upstream, for example in the `formatChangeList` method.